docs/topics/files.txt

```diff
@@ -181,3 +181,31 @@ For example, the following code will store uploaded files under
 :doc:`Custom storage systems </howto/custom-file-storage>` work the same way:
 you can pass them in as the ``storage`` argument to a
 :class:`~django.db.models.FileField`.
+
+Callable file storage
```

Comment:
Is this quite right? 

It's not that the file storage is callable but that the callable provides the storage... Maybe just "Using a callable" or "...to specify the storage"? 

(Feels a bit nit-picky but...)

Reply:
I can change the verbage there. Github is having some issues right now - https://status.github.com/messages